Header header

uint32 num_cluster
bool mirror_exists

pcl_msgs/PointIndices[] cluster_indices
sensor_msgs/PointCloud2 pointCloud
sensor_msgs/PointCloud2[] mirrorCloud
